dev/tasks/linux-packages/travis.linux.arm64.yml

```diff
@@ -131,7 +131,7 @@ script:
        echo "Name-Real: Test"; \
        echo "Name-Email: test@example.com"; \
        echo "%no-protection") | \
-        gpg --full-generate-key --batch
+       gpg --full-generate-key --batch
```

Comment:
Could you keep this indentation?
It's for piped `gpg`.

Reply:
Sorry, I missed the pipe sign.